three-cards-divination/pom.xml

```diff
@@ -16,5 +16,13 @@
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
**Suggestion:** The `mockito-core` dependency is already declared in the parent `pom.xml`. It's not necessary to declare it again in the module's `pom.xml`. [maintainability]
```suggestion

```